desktop/apps/consignments2/components/step_marker/index.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import block from 'bem-cn'
+import { connect } from 'react-redux'
+
+function StepMarker ({ currentStep, steps }) {
+  const b = block('consignments2-step-marker')
+
+  return (
+    <div className={b()}>
+      <div className={b('steps')}>
+        {
+          steps.map((step, index) => {
+            const active = (index <= currentStep)
+            return (
+              <div className={b('step').state({ active })} key={step.label}>
+                <div className={b('dot')} />
+                <div className={b('label')}>
+                  {step.label}
+                </div>
+              </div>
+            )
+          })
+        }
+      </div>
+    </div>
+  )
+}
+
+const mapStateToProps = (state) => ({
+  currentStep: state.submissionFlow.currentStep,
+  steps: state.submissionFlow.steps
```

Comment:
I recommend using selectors to avoid referring to the raw `state` throughout the view layer. It's small right now, but I think it's going to get unwieldy as both the state and the view layer grow.